contracts/test/integration/vip/vip6.sol

```diff
@@ -30,12 +32,18 @@ contract vip6 is DSTest, IVIP {
     }
 
     function mainnetSetup() public override {
+        if (false) {
+            ignore = false;
+        }
```

Comment:
Why do we do this?

Reply:
so that we don't get an error that this function could be pure.